MIGRATION-FROM-SCK.md

```diff
@@ -0,0 +1,325 @@
+* otelcol_otelsvc_k8s_ip_lookup_miss
+* otelcol_otelsvc_k8s_namespace_added
+* otelcol_otelsvc_k8s_namespace_updated
+* otelcol_otelsvc_k8s_pod_added
+* otelcol_otelsvc_k8s_pod_table_size
+* otelcol_otelsvc_k8s_pod_updated
+* otelcol_process_cpu_seconds
+* otelcol_process_memory_rss
+* otelcol_process_runtime_heap_alloc_bytes
+* otelcol_process_runtime_total_alloc_bytes
+* otelcol_process_runtime_total_sys_memory_bytes
+* otelcol_process_uptime
+* otelcol_processor_accepted_log_records
+* otelcol_processor_accepted_metric_points
+* otelcol_processor_batch_batch_send_size_bucket
+* otelcol_processor_batch_batch_send_size_count
+* otelcol_processor_batch_batch_send_size_sum
+* otelcol_processor_batch_timeout_trigger_send
+* otelcol_processor_dropped_log_records
+* otelcol_processor_dropped_metric_points
+* otelcol_processor_refused_log_records
+* otelcol_processor_refused_metric_points
+* otelcol_receiver_accepted_metric_points
+* otelcol_receiver_refused_metric_points
+* otelcol_scraper_errored_metric_points
+* otelcol_scraper_scraped_metric_points
+* scrape_duration_seconds
+* scrape_samples_post_metric_relabeling
+* scrape_samples_scraped
+* scrape_series_added
+* system.cpu.load_average.15m
+* system.cpu.load_average.1m
+* system.cpu.load_average.5m
+* system.cpu.time
+* system.disk.io
+* system.disk.io_time
+* system.disk.merged
+* system.disk.operation_time
+* system.disk.operations
+* system.disk.pending_operations
+* system.disk.weighted_io_time
+* system.filesystem.inodes.usage
+* system.filesystem.usage
+* system.memory.usage
+* system.network.connections
+* system.network.dropped
+* system.network.errors
+* system.network.io
+* system.network.packets
+* system.paging.faults
+* system.paging.operations
+* system.paging.usage
+* system.processes.count
+* system.processes.created
+* up
+
+### Changes in Objects in Splunk OpenTelemetry Connector for Kubernetes (SCK-OTEL)
+This chart currently does not support the collection of Kubernetes objects. This is future functionality. You can still use SCK to collect other objects with current functionality.
+
+## Migration Overview
+### Migration Options/Matrix
+ The following is a matrix for migrating from SCK to Splunk OpenTelemetry Collector for Kubernetes:
+
+| Method  | Logs | Metrics | Objects |
+|---|---|---|---|
+| SCK | Yes | Yes | Yes |
+| Splunk OpenTelemetry Collector for Kubernetes | Yes | Yes | No |
+
+As shown above, you can acquire logs and metrics using Splunk OpenTelemetry Collector for Kubernetes. If you have objects currently deployed and need objects data, we recommend you leave your current SCK objects deployment as-is (helm command args --set="splunk-kubernetes-logging.enabled=false,splunk-kubernetes-metrics.enabled=false,splunk-kubernetes-objects.enabled=true").  In this way, you can use the SCK just for objects data together with Splunk OpenTelemetry Collector for Kubernetes for logs and other telemetry data. 
+### Checkpoint Translation
```

Comment:
"Checkpoint translation"

"With..."

Can you use another phase besides "baton handoff"? This phrase may not be understood by all users.

Helm should always be capitalized.

"If you have properly configured the values.yaml file ..."

"pick up where it left off" should be changed to something like "continue working without interruption" or whatever the equivalent is. Again, "pick up where it left off" may not be understood by all users.